Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1859,11 +1865,13 @@ extension BrowserViewController: TabManagerDelegate {
             topTabsDidChangeTab()
         }
 
-        updateInContentHomePanel(selected?.url as URL?)
+        updateInContentHomePanel(selected?.url as URL?, didSelectTab: true)
         if let tab = selected, NewTabAccessors.getNewTabPage(self.profile.prefs) == .blankPage {
             if tab.url == nil, !tab.restoring {
+                navigationToolbar.updateIsSearchStatus(true)
```

Comment:
`updateIsSearchStatus` -> the naming of this function is confusing.
Lets use `buttonTypeOfSearchReload(to: .searchButton)`